src/assets/custom/css/_sass/_training-hero.scss

```diff
@@ -0,0 +1,150 @@
+.training-hero__background-image {
+  background-size: cover;
+  opacity: 0.9;
+  position: relative;
+  width: 100%;
+
+  @include small {
+    height: 216.27px;
+  }
+
+  @include medium-and-large {
+    height: 433px;
+  }
+
+  @include extra-large {
+    height: 519.5px;
+  }
+}
+
+.training-hero__course-instructor-avatar {
```

Comment:
Should this be moved down to be with the other -course-instructor- classes? :)

Reply:
Probably yes :D moved now 🎊